### PR TITLE
Alasdair/publish 03 24 2016

### PR DESCRIPTION
**Pull request by AlasdairSwan**
@andy-armstrong @dsjen please review - this is around https://openedx.atlassian.net/browse/FEDX-124. I think there may have been an issue with the previous PR I created not actually including the updates.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,6 +33,7 @@ nbproject
 .sass-cache
 *.map
 _site
+_tmp_preview_config.yml
```
Comment (dsjen) [inline .gitignore]: I think I'm out of the loop.  What is this file?

Comment (AlasdairSwan) [inline .gitignore]: I think this is to generate the S3 sandbox version, is that correct @andy-armstrong ?

Comment (dsjen) [inline .gitignore]: Ah!  I'm mostly just curious. :)

**Comment by dsjen** [comment]
LGTM!

File: .gitignore (hunk 1 above)
Comment (andy-armstrong) [inline .gitignore]: Yes, I had to create a temporary configuration file to get the preview to work. Not ideal, but oh well.

**Comment by andy-armstrong** [comment]
:+1: Fingers crossed that this will fix it! Is there anything you can add to the wiki page to clarify the process or to help with troubleshooting?

**Comment by AlasdairSwan** [comment]
@andy-armstrong assuming it fixes it I can add something around making sure the changes are actually present in the PR and about checking the settings tab to ensure the build was successful (in this case it was successful, it was just not the correct code).

**Comment by AlasdairSwan** [comment]
@andy-armstrong the build worked, and it updated ux.edx.org pretty much instantly.

**Comment by dsjen** [comment]
:balloon: congrats!!!  I see it...so pretty.

**Pull request merged**